packages/polaris-viz-core/src/constants.ts

```diff
@@ -9,6 +9,8 @@ import {InternalChartType, ChartState, Hue} from './types';
 export const LINE_HEIGHT = 14;
 export const SMALL_CHART_HEIGHT = 125;
 export const FONT_SIZE = 11;
+export const MOBILE_FONT_SIZE = 14;
```

Comment:
Should we call this touch device size, since it would also apply to tablets, with the way the check currently works?